tests/ha/multi_pod_failure/test_multi_server_pod_failure.py

```diff
@@ -2073,14 +2069,13 @@ def test_reads_during_kserver_pods_down(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-40572")
-    def test_reads_writes_during_kserver_pods_down(self):
+    def test_reads_writes_during_kserver_pods_fail(self):
         """
-        This test tests continuous READs/WRITEs while pods are failing till K
-        server pods  down - unsafe shutdown
+        This test tests READs/WRITEs while pods are failing during K server pods down - unsafe
+        shutdown
         """
-        LOGGER.info("STARTED: Test to verify continuous READs/WRITEs while %s (K) server pods "
-                    "were going down.", self.kvalue)
-
+        LOGGER.info("STARTED: Test to verify READs/WRITEs while %s (K) server pods were going "
```

Comment:
so no 2 parallel threads but just one thread of write and read, then it might not have 2 processes in parallel.